### PR TITLE
add case studies herigage impact

### PR DESCRIPTION
**Pull request by lucienicolier**

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[[source]]
```
Comment (m-kuhn) [inline Pipfile:1]: Is this file related to this PR (or added on purpose)?

File: docs/background/case-studies/heritage-impact-assessment.md [hunk 2]
```diff
@@ -0,0 +1,489 @@
...
+title: Heritage Impact Assessment
+---
+
+# Heritage Impact Assessment using QField
+
+## From Qgis to Qfield and Vice Versa: How the New Android Application Is Facilitating the Work of the Archaeologist in the Field
+
+**Roberto Montagnetti 1,* and Giuseppe Guarino 2**
+
+<sub><sup>1 adArte s.r.l. Archeologia, Restauro, ICT, 47921 Rimini, Italy</sup></sub>
```
Comment (m-kuhn) [inline docs/background/case-studies/heritage-impact-assessment.md:11]: I guess the `<sub><sup>` is here on purpose?

File: docs/background/case-studies/heritage-impact-assessment.md [hunk 3]
```diff
@@ -0,0 +1,489 @@
...
+ological jobs. In particular this article provides examples to use Qfield in open area excavation,
+Archaeological survey and impact assessment (HIA) projects.</sup></sub>
+
+<sub><sup>**Keywords:** Qfield; archeology; VIARCH; HIA; Qgis</sup></sub>
+
+----------------------------------------------------------------------------------------------------------------------
+
+### 1. Introduction
+
+The aim of this paper is to highlight the main benefits of using the Qfield app. An
```
Comment (m-kuhn) [inline docs/background/case-studies/heritage-impact-assessment.md:31]: Can we have a consistent spelling of "QField" and "QGIS" (capitalisation)?

**Comment by m-kuhn** [comment]
Nice new case study, thanks a lot to !

File: docs/background/case-studies/heritage-impact-assessment.md (hunk 2 above)
Comment (lucienicolier) [reply docs/background/case-studies/heritage-impact-assessment.md:11]: It's to make the text smaller. Do you have a better solution?

File: docs/background/case-studies/heritage-impact-assessment.md (hunk 3 above)
Comment (lucienicolier) [reply docs/background/case-studies/heritage-impact-assessment.md:31]: I took the original text, I didn't want to change it. But if you think it's fine to do it, I will.

Comment (m-kuhn) [reply docs/background/case-studies/heritage-impact-assessment.md:31]: I think it's ok to adjust it

File: docs/background/case-studies/heritage-impact-assessment.md (hunk 2 above)
Comment (m-kuhn) [reply docs/background/case-studies/heritage-impact-assessment.md:11]: `<small>` would be an option, but if you have put it here on purpose, that's also ok. Just wanted to be sure it's not a leftover.

File: Pipfile (hunk 1 above)
Comment (lucienicolier) [reply Pipfile:1]: It should not be here, what can I do?

Comment (m-kuhn) [reply Pipfile:1]: image

Comment (m-kuhn) [reply Pipfile:1]: CC @marioba do you have an idea how this could happen and what could/should be done to avoid this?

File: docs/background/case-studies/heritage-impact-assessment.md (hunk 2 above)
Comment (marioba) [reply docs/background/case-studies/heritage-impact-assessment.md]: Please use footnotes like here https://opengisch.github.io/QField-docs/background/case-studies/vanilla-survey/

See https://squidfunk.github.io/mkdocs-material/reference/footnotes/#usage for usage

File: Pipfile (hunk 1 above)
Comment (marioba) [reply Pipfile]: @m-kuhn, I think that's because @lucienicolier used pipenv locally and commited the generated pipfile by mistake. I'll add it to .gitignore
